### PR TITLE
Release notes for WCF CS in VS2017 15.5 Preview 1

### PR DESCRIPTION
**Pull request by zhenlan**
@dotnet-bot skip ci please

### CODE REVIEW

File: release-notes/WCF-Web-Service-Reference-notes.md [hunk 1]
```diff
@@ -1,16 +1,17 @@
...
 ## Release History
 
-### 1.0.0 - Visual Studio 2017 [v15.5 Preview 1]
+### 10/11/2017 - Visual Studio 2017 v15.5 Preview 1.0 [27009.1.d15rel]
 
-* [Supported features](https://github.com/dotnet/wcf/blob/master/release-notes/SupportedFeatures-v2.0.0.md)
+* WCF Web Service Reference connected services provider is included directly inside VS 2017 starting this release.
+* This tool is now available for all the languages that VS 2017 supports.
+* Added support for .NET Core projects that target latest .NET Framework and future releases with TargetFramework moniker (TFM) like net47, net471 etc.
+* Bug fixes (#2172, #2234, and more)
```
Comment (mlacouture) [inline release-notes/WCF-Web-Service-Reference-notes.md]: any way to add links to the bug numbers?

File: release-notes/WCF-Web-Service-Reference-notes.md [hunk 2]
```diff
@@ -1,16 +1,17 @@
-# WCF Web Service Reference Provider
+# WCF Web Service Reference Provider - Release Notes
 
-## Information
-
-* [Usage guide](/Documentation/WCF-Web-Service-Reference-guide.md)
-* [Report an issue](https://github.com/dotnet/wcf/issues/new)
+WCF Web Service Reference connected services provider is now part of Visual Studio 2017. Please refer to [usage guide](/Documentation/WCF-Web-Service-Reference-guide.md)
+for instructions or [report an issue or give us feedback](https://github.com/dotnet/wcf/issues/new).
```
Comment (mlacouture) [inline release-notes/WCF-Web-Service-Reference-notes.md]: for instructions, or [to report an issue, or to give us feedback]
// but you may want to have a native English speaker review this.

File: release-notes/WCF-Web-Service-Reference-notes.md [hunk 3]
```diff
@@ -1,16 +1,17 @@
-# WCF Web Service Reference Provider
+# WCF Web Service Reference Provider - Release Notes
 
-## Information
-
-* [Usage guide](/Documentation/WCF-Web-Service-Reference-guide.md)
-* [Report an issue](https://github.com/dotnet/wcf/issues/new)
+WCF Web Service Reference connected services provider is now part of Visual Studio 2017. Please refer to [usage guide](/Documentation/WCF-Web-Service-Reference-guide.md)
```
Comment (mlacouture) [inline release-notes/WCF-Web-Service-Reference-notes.md]: Please refer to [the usage guide]

File: release-notes/WCF-Web-Service-Reference-notes.md [hunk 4]
```diff
@@ -1,16 +1,17 @@
...
+for instructions or [report an issue or give us feedback](https://github.com/dotnet/wcf/issues/new).
 
 ## Release History
 
-### 1.0.0 - Visual Studio 2017 [v15.5 Preview 1]
+### 10/11/2017 - Visual Studio 2017 v15.5 Preview 1.0 [27009.1.d15rel]
 
-* [Supported features](https://github.com/dotnet/wcf/blob/master/release-notes/SupportedFeatures-v2.0.0.md)
+* WCF Web Service Reference connected services provider is included directly inside VS 2017 starting this release.
```
Comment (mlacouture) [inline release-notes/WCF-Web-Service-Reference-notes.md]: included as part of VS ...
// 'included directly inside' sounds a bit redundant to me.

File: release-notes/WCF-Web-Service-Reference-notes.md (hunk 1 above)
Comment (zhenlan) [reply release-notes/WCF-Web-Service-Reference-notes.md]: These are supposed to be links on GitHub automatically. Maybe not for MD. Anyway, I make them links.

File: release-notes/WCF-Web-Service-Reference-notes.md (hunk 3 above)
Comment (zhenlan) [reply release-notes/WCF-Web-Service-Reference-notes.md]: Updated.

File: release-notes/WCF-Web-Service-Reference-notes.md (hunk 2 above)
Comment (zhenlan) [reply release-notes/WCF-Web-Service-Reference-notes.md]: Yes, I know this paragraph is bit weird. I have it rewritten.

File: release-notes/WCF-Web-Service-Reference-notes.md (hunk 4 above)
Comment (zhenlan) [reply release-notes/WCF-Web-Service-Reference-notes.md]: Updated.

**Comment by zhenlan** [comment]
@StephenBonikowsky can you please help to proofread?

File: release-notes/WCF-Web-Service-Reference-notes.md [hunk 5]
```diff
@@ -1,16 +1,16 @@
...
-* [Report an issue](https://github.com/dotnet/wcf/issues/new)
+WCF Web Service Reference connected services provider is now part of Visual Studio 2017. Instructions can be found in the [usage guide](/Documentation/WCF-Web-Service-Reference-guide.md) document. Please [let us know](https://github.com/dotnet/wcf/issues/new) if you run into any issues or have any feedback.
 
 ## Release History
 
-### 1.0.0 - Visual Studio 2017 [v15.5 Preview 1]
+### 10/11/2017 - Visual Studio 2017 v15.5 Preview 1.0 [27009.1.d15rel]
 
-* [Supported features](https://github.com/dotnet/wcf/blob/master/release-notes/SupportedFeatures-v2.0.0.md)
+* WCF Web Service Reference connected services provider is included in VS 2017 starting this release.
```
Comment (StephenBonikowsky) [inline release-notes/WCF-Web-Service-Reference-notes.md]: Insert "with" between starting and this.

**Comment by StephenBonikowsky** [comment]
One nit, otherwise LGTM

File: release-notes/WCF-Web-Service-Reference-notes.md (hunk 5 above)
Comment (zhenlan) [reply release-notes/WCF-Web-Service-Reference-notes.md]: Fixed. Thanks @StephenBonikowsky.

**Pull request merged**